test/parallel/test-worker-eventlooputil.js

```diff
@@ -104,8 +105,9 @@ function checkWorkerActive() {
     const w2 = workerELU(w);
 
     assert.ok(w2.active >= 50, `${w2.active} < 50`);
-    assert.ok(idleActive(wElu) > idleActive(w2),
-              `${idleActive(wElu)} <= ${idleActive(w2)}`);
+    assert.ok(wElu.active >= 50, `${wElu.active} < 50`);
+    assert.ok(idleActive(wElu) < idleActive(w2),
```

Comment:
@Flarna - can you explain why `wElu` is used instead of `w2`? according to me, `w2` looks correct, as that holds the delta utilization data between the spin time, and that is exactly what we are measuring?

Reply:
`wElu` is measured inside the worker, `w2` is the same + the messaging overhead measure on main thread.
Therefore `wElu` must be smaller then `w2` but both must show that worker was active for at least the spin time.